### PR TITLE
Fix signing not working using Ledger on Testnet

### PR DESCRIPTION
**Pull request by binier**
Mainnet `networkId` was being used instead of dynamic(in this case testnet), which made created signatures invalid on testnet.

### CODE REVIEW

**Comment by wjdfx** [comment]
Thanks for your controbute, it's not only `networkId` field problem.
We are trying to solve this problem: https://discord.com/channels/484437221055922177/733755408161833040/988987213956149268

**Comment by binier** [comment]
@wjdfx What else is there? I didn't see anything else on discord.

Note that I tried this fix and ledger signing works on testnet now.

**Comment by wjdfx** [comment]
> @wjdfx What else is there? I didn't see anything else on discord.
> 
> Note that I tried this fix and ledger signing works on testnet now.

Oh, If you have been tried and worked, let me try again, before we use same change but it doesn't worked.

**Comment by wjdfx** [comment]
@binier are you sure it's will successfully after this code? You didn't got `[Invalid_signature] ` error?

**Comment by binier** [comment]
https://devnet.minaexplorer.com/wallet/B62qqXi8mm3UsY45DJFpEc6KK4k69hcVmWf6xm4FRYcuUKXUGW2BGit

@wjdfx The second transaction was done using Ledger + locally built Auro wallet.

Can you try clean/rebuilding the extension and adding to chrome? Maybe it's a cache thing..

**Comment by wjdfx** [comment]
@binier can you add my discord: niuniu | Bit Cat#9001

Or telegram @sanxiacat 

Thank you.
